src/edit_view.rs

```diff
@@ -137,13 +145,23 @@ impl Widget for EditView {
         if let Some(cmd) = payload.downcast_ref::<EditViewCommands>() {
             match cmd {
                 EditViewCommands::ViewId(view_id) => {
-                    self.view_id = view_id.to_string();
+                    self.view_id = Some(view_id.to_string());
+                    self.viewport = 0..0; // zorch viewport
```

Comment:
I think maybe we want to actually call `update_viewport` as well; I can see this mostly working, as it's very likely that it will be called in the future, but as we make `request_layout` more lazy, that's less guaranteed.